import java.util.Scanner;
import java.lang.Exception;

public class calc
{	public static void main(String args[])
	{	try
		{	int a,b,ch,res;
			Scanner obj=new Scanner(System.in);
			System.out.println("Enter two nos. :");
			a=Integer.parseInt(args[0]);
			b=Integer.parseInt(args[1]);
			System.out.println("Press 1 for Addition");
			System.out.println("Press 2 for Substraction");
			System.out.println("Press 3 for Multiplication");
			System.out.println("Press 4 for Division");
			System.out.println("Enter your choice :");
			ch=obj.nextInt();

			switch(ch)
			{	case 1: res=a+b;
					System.out.println("Addition :"+res);
					break;
				case 2: res=a-b;
					System.out.println("Substraction :"+res);
					break;
				case 3: res=a*b;
					System.out.println("Multiplication :"+res);
					break;
				case 4: res=a/b;
					System.out.println("Division :"+res);
					break;
				default: break;
			}
		}
		catch(NumberFormatException nfe)
		{	System.err.println("Not an integer type !!");
		}
		catch(ArithematicException e1)
		{	System.err.println("Divide by zero exception !!");
		}
	}
}
		
